docs/modules/liberty.md

```diff
@@ -0,0 +1,56 @@
+# Liberty Containers
```

Comment:
In order to display the docs we should add a section `Application Servers` just like we have it for databases

https://github.com/testcontainers/testcontainers-java/blob/8465795ab9067447ca8b7d3e307cadcec8de377a/mkdocs.yml#L48-L49